gap/profiling.gi

```diff
@@ -938,23 +938,23 @@ function(data, outfile, pathtoremove, extraargs...)
     IO_Close(outstream);
 end);
 
+# Use a temporary check to support GAP versions without ARCH_IS_WSL
+__profiling_pkg_temp_wsl_check := function()
+local bash, val;
+if IsBoundGlobal("ARCH_IS_WSL") then
+  return ValueGlobal("ARCH_IS_WSL")();
+fi;
+bash := Filename(DirectoriesSystemPrograms(), "bash");
+if bash = fail then
+  return false;
+fi;
+val := Process(Directory("/"), bash, InputTextNone(), OutputTextNone(), ["which", "explorer.exe"]);
+return val = 0;
+end;
+
```

Comment:
You could also be brutal and just import the code for `ARCH_IS_WSL` if it is missing:
```gap
if not IsBound(ARCH_IS_WSL) then
BIND_GLOBAL("ARCH_IS_WSL",function()
  # There is no official way to detect this, but this method is
  # suggested: https://github.com/microsoft/WSL/issues/423
  # Note that at some point the string was changed from 'Microsoft' to
  # 'microsoft'.
  return ARCH_IS_UNIX() and IsBound(GAPInfo.KernelInfo.uname.release) and (
    POSITION_SUBSTRING(GAPInfo.KernelInfo.uname.release, "Microsoft", 0) <> fail or
    POSITION_SUBSTRING(GAPInfo.KernelInfo.uname.release, "microsoft", 0) <> fail);
end);
fi;
```

Reply:
I can't do this (just checked) because I added KernelInfo.uname.release at the same time as ARCH_IS_WSL, so this would only work in GAPs where the function is already present anyway :)